storage/test/acl.test.js

```diff
@@ -0,0 +1,379 @@
+      assert.equal(sample.mocks.bucket.acl.delete.firstCall.args.length, 2, 'add received 2 arguments');
+      assert.deepEqual(sample.mocks.bucket.acl.delete.firstCall.args[0], {
+        entity: entity
+      }, 'get received options');
+      assert(callback.calledOnce, 'callback called once');
+      assert.equal(callback.firstCall.args.length, 1, 'callback received 1 argument');
+      assert.ifError(callback.firstCall.args[0], 'callback did not receive error');
+      assert(console.log.calledWith('Deleted access controls from: gs://%s', bucketName, ''));
+    });
+    it('should delete an entity\'s "default" access controls from a bucket', function () {
+      var sample = getSample();
+      var callback = sinon.stub();
+      var options = {
+        bucket: bucketName,
+        entity: entity,
+        default: true
+      };
+      sample.mocks.bucket.acl.default.delete.callsArgWith(1, null, sample.mocks.aclObject);
+
+      sample.program.deleteAccessControl(options, callback);
+
+      assert(sample.mocks.bucket.acl.default.delete.calledOnce, 'add called once');
+      assert.equal(sample.mocks.bucket.acl.default.delete.firstCall.args.length, 2, 'add received 2 arguments');
+      assert.deepEqual(sample.mocks.bucket.acl.default.delete.firstCall.args[0], {
+        entity: entity
+      }, 'get received options');
+      assert(callback.calledOnce, 'callback called once');
+      assert.equal(callback.firstCall.args.length, 1, 'callback received 1 argument');
+      assert.ifError(callback.firstCall.args[0], 'callback did not receive error');
+      assert(console.log.calledWith('Deleted access controls from: gs://%s', bucketName, ''));
+    });
+    it('should delete access controls from a file', function () {
+      var sample = getSample();
+      var callback = sinon.stub();
+      var options = {
+        bucket: bucketName,
+        file: fileName,
+        entity: entity
+      };
+
+      sample.program.deleteAccessControl(options, callback);
+
+      assert(sample.mocks.file.acl.delete.calledOnce, 'add called once');
+      assert.equal(sample.mocks.file.acl.delete.firstCall.args.length, 2, 'delete received 2 arguments');
+      assert(callback.calledOnce, 'callback called once');
+      assert.equal(callback.firstCall.args.length, 1, 'callback received 1 argument');
+      assert.ifError(callback.firstCall.args[0], 'callback did not receive error');
+      assert(console.log.calledWith('Deleted access controls from: gs://%s/%s', bucketName, fileName));
+    });
+    it('should handle error', function () {
+      var error = 'error';
+      var sample = getSample();
+      var callback = sinon.stub();
+      var options = {
+        bucket: bucketName
+      };
+      sample.mocks.bucket.acl.delete = sinon.stub().callsArgWith(1, error);
+
+      sample.program.deleteAccessControl(options, callback);
+
+      assert(callback.calledOnce, 'callback called once');
```

Comment:
I would say the two main benefits are:

1) Use of `sinon.stub` as the callback allows for additional assertions about the code execution (it gathers more info about how the callback is used)
2) The test function now actually looks synchronous in addition to being synchronous.